pkg/storage/article_storage.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"news-aggregator/pkg/model"
+	"time"
+)
+
+//go:generate mockgen -destination=mocks/mock_article.go -package=mocks news-aggregator/pkg/storage ArticleStorage
+
+// ArticleStorage is an interface that defines the methods for interacting with the article storage.
```

Comment:
i understand this is an interface, no need to duplicate this. Try to answer below questions:
1. What are the key responsibilities of this interface? 
2. What are the expected behaviors or guarantees?
3. What are the common errors or exceptions, and how should they be handled?
4. What are the known limitations or restrictions?
5. Are there any usage guidelines or best practices?